drivers/at86rf215/include/at86rf215_internal.h

```diff
@@ -49,6 +49,9 @@ extern "C" {
 /** in 802.15.4 oqpsk each symble is 4 bits, not about the others */
```

Comment:
```suggestion
/** in 802.15.4 oqpsk each symbol is 4 bits, not about the others */
```

Reply:
I removed the define as it's not needed anymore.